src/redux/reducers/errorHandler.js

```diff
@@ -12,8 +9,8 @@ export default (state = initialState, action) => {
       };
     }
     case CLEAR_ERROR_MSG: {
```

Comment:
There is no braces after the case in as switch statement
```
switch(action.type){
  case type:
     ...
  default:
     ....
}
```

Reply:
Resolved!